NuRadioMC/utilities/inelasticities.py

```diff
@@ -13,8 +20,12 @@
 cspeed = constants.c * units.m / units.s
 G_F = constants.physical_constants['Fermi coupling constant'][0] * units.GeV ** (-2)
 
+nu_energies_ref, yy_ref, flavors_ref, ncccs_ref, dsigma_dy_ref = pickle.load(lzma.open(os.path.join(os.path.dirname(__file__), "data", "BGR18_dsigma_dy.xz")))
+ncccs_ref = np.array(ncccs_ref)
 
-def get_neutrino_inelasticity(n_events, rnd=None):
+
+def get_neutrino_inelasticity(n_events, model="CTW", rnd=None,
+                              nu_energies=1 * units.EeV, flavors=12, ncccs="CC"):
```

Comment:
Not a huge fan of the nccc ccnc inttype interaction_type channel at various places in NuRadioMC (generator/inelasticieite/cross_sections) ... maybe that is because there is no official nomenclature... will it get worse when someone adds "res"/"glashow" in the future?

Reply:
But probably not the task of this PR... :)

Reply:
I agree, do you have a better suggestion?

Reply:
not really... I'd lean towards using `interaction_type` but just because I `ncccs` would make me feel I fell asleep pressing the c key... but I see the advantage that nccc tells everyone directly what this variable describes... You may leave it as is